### PR TITLE
Add DFGM service

### PR DESCRIPTION
**Pull request by DanielSacro**

### CODE REVIEW

**Review by joshdellaz — APPROVED** [review]

File: main/main.c [hunk 1]
```diff
@@ -58,10 +58,10 @@
 #include "eps.h"
 #include "sband.h"
 #include "system.h"
+#include "dfgm.h"
 
 #include "sband_binary_tests.h"
 #include "deployablescontrol.h"
-#include "HL_system.h"
```
Comment (Taylorr82) [inline main/main.c:64]: Why is this removed?

**Review by Taylorr82 — CHANGES_REQUESTED** [review]

File: main/main.c (hunk 1 above)
Comment (DanielSacro) [reply main/main.c:64]: It was already included earlier on line 36 as <HL_system.h>

**Review by Taylorr82 — APPROVED** [review]
